src/index.js

```diff
@@ -10,17 +10,16 @@ const Actions = Object.freeze({
 const getBrowserEnv = () => {
   // Attempt to honor this environment variable.
   // It is specific to the operating system.
-  // See https://github.com/sindresorhus/opn#app for documentation.
+  // See https://github.com/sindresorhus/open#app for documentation.
   const value = process.env.BROWSER;
-  let action;
+  let action = Actions.BROWSER;
   if (!value) {
     // Default.
     action = Actions.BROWSER;
   } else if (value.toLowerCase() === 'none') {
     action = Actions.NONE;
-  } else {
```

Comment:
CRA also takes care in case it runs on node scripts as you can see [here](https://github.com/facebook/create-react-app/blob/d2de54b25cc25800df1764058997e3e274bd79ac/packages/react-dev-utils/openBrowser.js#L46). I left all that part out because I imagined there was a reason you didn't have it. Wanted to know your point of view about it!

Reply:
> CRA also takes care in case it runs on node scripts as you can see [here](https://github.com/facebook/create-react-app/blob/d2de54b25cc25800df1764058997e3e274bd79ac/packages/react-dev-utils/openBrowser.js#L46). I left all that part out because I imagined there was a reason you didn't have it. Wanted to know your point of view about it!

Some background story, https://github.com/gatsbyjs/gatsby/pull/6550

I guess my intention was the `script` option is not a use case in `gatsby`, so I ended up excluding it. Also, I wasn't expecting people besides `gatsby` who would actually use this library.

I don't mind adding the `script` option back to `better-opn`, it should be done in a separate PR tho.